### PR TITLE
Mid Term chole

### PR DESCRIPTION
**Pull request by fifo36**

### CODE REVIEW

File: week09/124. Binary Tree Maximum Path Sum.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        int L = getMaxPathSum(root->left);
+        int R = getMaxPathSum(root->right);
+        maxSum = max(maxSum, L + R + root->val);
+
+        return max(0, max(L, R) + root->val);
+
+    }
+
+    int maxPathSum(TreeNode* root) {
+        maxSum = -1000000;
```
Comment (Sadman007) [inline week09/124. Binary Tree Maximum Path Sum.cpp:18]: `INT_MIN` is always safe. What if the lowest value in the tree node is less than -10^6?

**Review by Sadman007 — APPROVED** [review]
LGTM

**Comment by fifo36** [comment]
closed

**Comment by fifo36** [comment]
closed

**Pull request closed without merging**